config/v3_2_experimental/types/schema.go

```diff
@@ -131,6 +131,7 @@ type Partition struct {
 	GUID               *string `json:"guid,omitempty"`
 	Label              *string `json:"label,omitempty"`
 	Number             int     `json:"number,omitempty"`
+	ResizePartition    *bool   `json:"resizePartition,omitempty"`
```

Comment:
Hmm, I wonder if we should just use `resize` to keep it shorter, there isn't anything else in the current spec that has a resize option.

cc @bgilbert

Reply:
The commit message still calls it `resizePartition`.